fera_full.py

```
@@ -1,3 +1,189 @@
+                        relats = cursor.fetchall()
+                        notindexed.append(pathp)
+                        for rel in relats:                                   
+                            notindexed.append(os.path.join(global_settings.pathdb.parent, rel[2]))
+                        if(len(notindexed)>0): 
+                            tupleinfo = indexador_fera.addrels('relatorio', pathpdfinput = notindexed, pathdbext=sys.argv[1], \
+                                                                   rootx=global_settings.root, sqliteconnx=sqliteconn)
+                    except sqlite3.OperationalError as ex:  
+                        notindexed.append(pathp)
+                        try:
+                            sqliteconn.close()
+                        except:
```

Ruff

Do not use bare `except`

(E722)